trackintel/analysis/__init__.py

```diff
@@ -0,0 +1 @@
+from .transport_mode_identification import predict_transport_mode
```

Comment:
is this consistent everywhere in trackintel. I have never included statements like this... maybe you'll give me a quick start on init statements at some point...

Reply:
I think it mostly depends on what you want to be exposed when importing the module. Like this, you can say `trackintel.analysis.predict_transport_mode`. I think the main functions within a module are often exposed like this. 

In addition, it prevents errors that a module (or rather a function defined in another file in the module folder) is not imported yet (I think). If you remove the line and let the tests run the `test_transport_mode_identification` test fails as the module is not imported anywhere. 

For the other tests it seems to work as they are (luckily (?)) imported along the way somewhere (most in `preprocessing` which in turn is imported in the `model` files, and those are imported in the `trackintel/__init__.py`).